src/install.rst

```diff
@@ -6,148 +6,241 @@ Installing Nextstrain
+         - This is necessary for a dependency of Nextstrain CLI and `we are investigating options to eliminate this step <https://github.com/nextstrain/cli/issues/31#issuecomment-970641263>`_.
 
-    .. code-block:: bash
+      3. Open an Anaconda Prompt, which can be found in the Start menu.
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-        conda activate nextstrain
-        nextstrain check-setup --set-default
+         .. code-block:: none
 
-    The final output from the last command should look like this:
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-    .. code-block:: bash
+      5. Install the remaining Nextstrain components by `installing Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+      6. Confirm that the installation worked.
 
-        Setting default environment to docker.
+         .. code-block:: none
 
-    Finally, download the latest Docker image for Nextstrain.
+            nextstrain check-setup --set-default
 
-    .. code-block:: bash
+         The final output from the last command should look like this:
 
-        nextstrain update
+         .. code-block:: none
 
-    Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Nextstrain Conda environment with ``conda activate nextstrain``.
-    Next, try :doc:`tutorials/quickstart`.
+            Setting default environment to docker.
 
-Upgrade an existing installation
-================================
+   .. group-tab:: WSL on Windows
 
-.. tabs::
+      .. note::
+
+         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for augur <augur:installation/installation>` and :doc:`install auspice via npm <auspice:introduction/install>`.
+
+      1. `Install WSL 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
+      2. Open a WSL terminal by running **wsl** from the Start menu.
+      3. Install Miniconda:
+
+         .. code-block:: bash
+
+            wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            bash Miniconda3-latest-Linux-x86_64.sh
+            # follow through installation prompts
+            rm Miniconda3-latest-Linux-x86_64.sh
+
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
+
+         .. code-block:: bash
+
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
+
+      5. Install the remaining Nextstrain components. There are two options:
+
+         a. Docker (recommended) – on Windows, `install Docker Desktop for WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+
+            - Make sure to follow through the last step of enabling **WSL Integration**.
+
+         b. Native – install all the necessary software using conda:
```

Comment:
Request from @j23414 [on Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1637685730181700?thread_ts=1637626719.175300&cid=C01LCTT7JNN):

> Any chance we could add a "mamba" tab (similar to the conda and docker tabs)? Or simply add mamba instructions to the conda tab ([link to notes](https://github.com/j23414/usingnextstrain#1-nextstrain-installation)).

Reply:
Updated to use mamba for all package installation commands: https://github.com/nextstrain/docs.nextstrain.org/pull/81/commits/af210cd12e62324435590b3c575ce7775aa20a05